### DISCUSSION TITLE
Purpur running start.sh

### DISCUSSION THREAD

**Discussion by Masonion** [PurpurMC/Purpur#1074] [__Dismissed__] [answered]
Hey, I've noticed that Purpur doesn't run start.sh but will run start.bat, wondering if this is something that can be added?

**Comment by BillyGalbreath** [comment]
What? Join us on Discord if you want to have a discussion.

**Comment by 12emin34** [comment]
you are running Windows, since it doesn't support shell scripts, only the bat file will work, this has nothing to do with Purpur tho.

**Comment by Masonion** [comment, marked as answer]
Worked out the issue, sorry I was misinformed.
